mkdocs/docs/HPC/troubleshooting.md

```diff
@@ -464,30 +464,32 @@ memory you request.
 ## Module conflicts
 
 Modules that are loaded together must use the same toolchain version or common dependencies. In the following
-example, we try to load a module that uses the `intel-2018a` toolchain
-together with one that uses the `intel-2017a` toolchain:
+example, we try to load a module that uses the `intel-2023a` toolchain
+together with one that uses the `intel-2022a` toolchain:
 
 ```bash
-$ module load Python/2.7.14-intel-2018a
-$ module load  HMMER/3.1b2-intel-2017a
-Lmod has detected the following error: A different version of the 'intel' module is already loaded (see output of 'ml'). 
-You should load another 'HMMER' module for that is compatible with the currently loaded version of 'intel'. 
-Use 'ml avail HMMER' to get an overview of the available versions.
+$ module load Python/3.12.3-GCCcore-13.3.0
+$ module load  Pillow/8.0.1-GCCcore-10.2.0
```

Comment:
Use more recent module here, `Pillow/10.0.0-GCCcore-12.3.0`?

Reply:
Fixed in [module conflict example with newer versions](https://github.com/hpcugent/vsc_user_docs/pull/710/commits/1ac539f4e285168ede4e970ecaf6ebc55072ca2f)